src/lib/bootloader/serial_console.rb

```diff
@@ -0,0 +1,101 @@
+require "yast"
+
+Yast.import "Arch"
+
+module Bootloader
+  # Represents parameters for console. Its main intention is easy parsing serial
+  # console parameters parameters for grub or kernel and generate it to keep it
+  # in sync.
+  class SerialConsole
+    PARITY_MAP = {
+      "n" => "no",
+      "o" => "odd",
+      "e" => "even"
+    }
+    SPEED_DEFAULT = 9600
```

Comment:
For consistency: SPEED_DEFAULT = "9600", **or** make WORD_DEFAULT a \[Integer, nil]